src/middlewares/user-validator.js

```diff
@@ -0,0 +1,15 @@
+import Validator from '../helpers/validator';
+import ErrorHandler from './error-handler';
+
+const userValidator = {
+  signUp: [
+    Validator.isValidEmail(),
+    Validator.isValidName('firstname'),
+    Validator.isValidName('lastname'),
+    Validator.isValidName('username'),
+    Validator.isValidPassword(),
+    (req, res, next) => ErrorHandler.validatorError(req, res, next)
```

Comment:
`ErrorHandler.validatorError` is already a middleware. You don't need to return it from the arrow function. You can simply pass it directly. e.g: 

```
signUp: [
    Validator.isValidEmail(),
    Validator.isValidName('firstname'),
    Validator.isValidName('lastname'),
    Validator.isValidName('username'),
    Validator.isValidPassword(),
    ErrorHandler.validatorError
]
```

Reply:
files are not named in camel case, it can cause issues, it is advisable to name them with either _ or - to separate words, e.g error-handler.js or error_handler.js

Reply:
for reference purpose, check your npm modules, they always follow this convention

Reply:
Alright, we can bring this up to the team and stick to one pattern. But kindly look into the first part of the comment.

Reply:
already done